semeio/workflows/spearman_correlation_job/job.py

```diff
@@ -99,6 +93,13 @@ def _calculate_correlation_matrix(data):
     return data.rank().corr(method="pearson")
 
 
-def _cluster_analysis(correlation_matrix, threshold, criterion, depth, method, metric):
+def _cluster_analysis(
```

Comment:
Maybe best to just inline it and avoid `**cluster_args`.

Reply:
In this specific case I would tend to agree, the reason I have chosen this approach is because the next PR will introduce a second clustering function, with completely different arguments which will need to be passed to this function. Am happy to take suggestions though.

Reply:
Alright then we will see how it looks later :)